noxfile.py

```diff
@@ -6,6 +6,19 @@
 import shutil
 import nox
 
+@nox.session(py="3")
+def translation(session):
+    session.install("-r", "requirements.txt")
+    target_dir = "locales"
+    session.run(
+        "sphinx-build", 
+        "-b",  # select a builder
+        "gettext",  # build gettext-style message catalogs (.pot file)
+        "source/",  # where the rst files are located
+        target_dir, # where to put the .pot file
+        "-d", 
+        ".nox/", # path to put the cache (.doctrees)
```

Comment:
*I think* this should be
```suggestion
        "-d", ".nox/.doctrees/", # path to put the cache
        target_dir, # where to put the .pot file
```
(plus putting target as the last positional arg to avoid confusion with options having values)

Reply:
I wonder if we could make a module-level constant tuple with shared args to reuse across all sessions. OTOH, this could be explored in a separate PR anyway.

Reply:
So maybe open another PR for this will be better? :)